### ISSUE TITLE
Spam with my name

### ISSUE THREAD

**Issue by TiagoDanin** [codetriage/codetriage#686]
Today I open GitHub and I see my name in ten more pull request
- https://github.com/search?o=desc&q=TiagoDanin&s=updated&type=Issues&utf8=%E2%9C%93
I approved nothing...

I deleted [my account](https://www.codetriage.com/users/32432) today 😔

Please remove these PRs:
- https://github.com/ShareX/ShareX/pull/3129
- https://github.com/primitivorm/latino/pull/119
- https://github.com/telegraf/telegraf/pull/330
- https://github.com/benbaptist/minecraft-wrapper/pull/524
- https://github.com/MinhasKamal/DownGit/pull/16
- https://github.com/PyGithub/PyGithub/pull/697
- https://github.com/LibreLabUCM/HALIX9000/pull/2
- https://github.com/devtranslate/devtranslate.github.io/pull/41
- https://github.com/Programeiros/TycotBot/pull/65
- https://github.com/RedisLabsModules/rejson/pull/57
- https://github.com/duckduckgo/zeroclickinfo-goodies/pull/4584
- https://github.com/jh0ker/mau_mau_bot/pull/51
- https://github.com/gulpjs/gulp/pull/2118

**Comment by schneems** [comment]
Sorry about that. I wasn't trying to spam you, I was trying to give context around how the issue got added to code triage. In the markdown you're not @-mentioned but rather it's a link to your profile. I guess github sends notifications anyway. I can change it to not use the @ in front of the name.

If you deleted your account you're good to go. No further action is required from you. I also manually closed all those PRs you linked to.

I'm manually selecting repos to send out PRs to, I didn't send any of your repos (starting with `
TiagoDanin/` since they don't have badges. I think you got hit hard because you added so many repos very quickly.

Sorry again about that, this is a new experiment. I got the idea from https://github.com/gitterHQ/readme-badger.
